cypress/e2e/Gui.cy.js

```diff
@@ -0,0 +1,63 @@
+describe("EngageSphere GUI", () => {
+  beforeEach(() => {
+    cy.setCookie('cookieConsent', 'accepted');
+    cy.visit("/");
+  });
```

Comment:
```suggestion
  });

```

Deixe linhas em branco entre o bloco `beforeEach` e o bloco `it` para facilitar diferenciar que acabou um e começou outro.

Veja um exemplo abaixo:

```js
describe('EngageSphere', () => {
  beforeEach(() => {
    cy.visit('/')
  })

  it('shows the default greeting (i.e., Hi, there...)', () => {
    // Test implementation here
  })

  it('shows the customized greeting (i.e., Hi, Joe...)', () => {
    // Test implementation here
  })

  // Outros casos de teste aqui...
})

```